src/client/actions/InviteCreate.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const Action = require('./Action');
+const Invite = require('../../structures/Invite');
+const { Events } = require('../../util/Constants');
+
+class InviteCreateAction extends Action {
+  handle(data) {
+    const client = this.client;
+    const inviteData = Object.assign(data, {
+      guild: client.guilds.get(data.guild_id),
+      channel: client.channels.get(data.channel_id),
+    });
+    const invite = new Invite(client, inviteData);
```

Comment:
According to hydra, this should check if the guild or channel is null before creating an Invite.

Ref: https://github.com/discordjs/discord.js/pull/3728/files/0843669a4bb790ff6e4734c1a02a9bfb72d94852#r368896512